src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> map = getPositionsFromFile(fromFileName);
+        int supply = map.get("supply");
+        int buy = map.get("buy");
+        int result = supply - buy;
+        writeToFile(supply, buy, result, toFileName);
+    }
+
+    private Map<String, Integer> getPositionsFromFile(String fromFileName) {
+        Map<String, Integer> map = new HashMap<String, Integer>();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            String[] dividedLine;
+            
+            while (line != null) {
+                dividedLine = line.split(",");
+                int number = Integer.parseInt(dividedLine[1]);
```

Comment:
Why wouldn't you get another local variable like you did for number:
String operationType = dividedLine[0];
And then work with it instead of dividedLine[0]. It will increase readability of code.